README.md

```diff
@@ -292,6 +292,10 @@ Makes it possible to use your MyAnimeList/Anilist/Kitsu/Simkl anime/mangalist as
                 <td><a href="https://www.animetoast.cc"><img src="https://favicon.malsync.moe/?domain=https://www.animetoast.cc"> Animetoast</a></td>
```

Comment:
this file is autogenerated on release please dont edit

Reply:
I reverted the file back to master and did the same for the other autogenerated ones.